bin/package_build.py

```diff
@@ -14,33 +14,33 @@ def purify(dirty):
 
 def debian():
 	global DATA, DATA_FILE_LOCATION
-	q = 'Debian_Buster_List.json'
-	file_name = f'{DATA_FILE_LOCATION}/{q}'
-	try:
-		req = requests.get(f"https://packages.debian.org/stable/allpackages")
-		data = req.text
-		if req.status_code == 404:
-			raise Exception("404 File not found")
-	except Exception as e:
-		print("Couldn't pull. Error: ",str(e))
-	else:
-		ref_data = data.split('<dl>')[1].split('</dl>')[0]
-#		cut unwanted unicode by placing regexes here. This is important
-		sp_reg = r' \[<.*\]'
-		if re.search(sp_reg, ref_data):
-			ref_data = re.sub(sp_reg, '', ref_data)
-		ref_data = re.sub(r"(<dt><a[^>]*'>)", '{\n\t"packageName": "', ref_data)
-		ref_data = re.sub(r"(<\/a>(( \()|(</dt>))+)", '",\n\t"version": "', ref_data)
-		ref_data = re.sub(r"\n*(<dd[^>]*>)", "%%<dd>", ref_data)
-		ref_data = re.sub(r"((\)<\/dt>)*\n*%%)", '",\n', ref_data)
-		ref_data = re.sub(r"(<dd[^>]*>)", '\t"description": "', ref_data)
-		ref_data = re.sub(r"(<\/dd>)",'"\n},',ref_data)
-#		To remove undecodable ascii chars
-		ref_data = purify(ref_data)
-		DATA = open(file_name, 'w')
-		DATA.write('['+ref_data+'{}\n]')
-		DATA.close()
-		print(f"Saved!\nfilename: {q}")
+	q = ['Debian_Buster_List.json', 'Debian_Bullseye_List.json']
+	urls = ['https://packages.debian.org/buster/allpackages', 'https://packages.debian.org/bullseye/allpackages']
```

Comment:
It's pretty obvious in other distributions because it's a different repo, or says something like "zsystems" or "s390x" in the directories, but for Debian it's not obvious. How are we pulling just the s390x packages? And can you leave a comment in the code for future contributors since it's not obvious?

Obviously not specific to Bullseye, I missed this when I reviewed for the initial PR for Debian support, oops!